src/types.ts

```diff
@@ -32,10 +32,10 @@ type Named<Key extends string, T> = { name: string } & Record<Key, T>;
 
 export interface SkycliConfig {
   api_version: string;
-  clusters?: Named<"cluster", ClusterConfig>[];
-  users?: Named<"user", ClusterUserConfig>[];
-  contexts?: Named<"context", ConfigContext>[];
-  current_context?: string;
+  clusters: Named<"cluster", ClusterConfig>[];
+  users: Named<"user", ClusterUserConfig>[];
```

Comment:
This is unsafe because we used to omit `users` key so in an existing config in the wild it could be nullish. Let's stick with the `?` here.